src/main/webapp/app/modules/home/filter-activity.tsx

```diff
@@ -84,19 +84,29 @@ export class FilterActivity extends React.Component<IFilterActivityProps, IFilte
   }
 
   getPartnerList = () => {
-    this.props.getPartnerList();
+    if (_.isEmpty(this.props.partnerList)) {
```

Comment:
Are all filters cleared when the user is logging out? If no, then I think we won't show filter options for his records, but the old ones.